logpyle/__init__.py

```diff
@@ -867,12 +887,13 @@ def add_quantity(self, quantity: LogQuantity, interval: int = 1) -> None:
         :param interval: interval (in time steps) when to gather this quantity.
         """
 
-        def add_internal(name, unit, description, def_agg):
+        def add_internal(name, unit, description, def_agg, set_value=None):
```

Comment:
Now that we're standardizing `set_value` in `PushLogQuantity`, I'm no longer sold on the value of passing in `set_value`.